src/Akeneo/Bundle/FrontofficeBundle/Resources/views/Default/index.html.twig

```diff
@@ -1 +1,110 @@
-Salut !
+{% extends '::base.html.twig' %}
+
+{% block body %}
+    <header class="grey lighten-2 z-depth-1">
+        <div id="content">
+            <h2>Let's meet up!</h2>
+
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+                et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                cillum dolore eu fugiat nulla pariatur.
+            </p>
+        </div>
+    </header>
+    <div id="add_event_button">
+        <a class="btn-floating btn-large waves-effect waves-light red z-depth-2 tooltipped" data-position="right"
+           data-delay="50" data-tooltip="Add your event!"><i class="material-icons"
+                                                             style="font-style: normal;">+</i></a>
+    </div>
+    <div id="map" class="z-depth-1"></div>
+
+    <div style="display:none">
+        <div id="event_wrapper" class="row z-depth-2">
+            <div class="col s3">
+                <div class="card event_info">
+                    <div class="card-content white-text">
+                        <img src="{{ gravatar_url('remy.betus@gmail.com') }}"/>
+
+                        <div id="name">Rémy Bétus - <strong>Akeneo</strong></div>
+
+                        <div id="email">
+                            <a href="mailto:remy.betus@gmail.com">remy.betus@gmail.com</a>
+                        </div>
+                        <div id="location">
+                            Le lieux unique, Nantes, France
+                        </div>
+                        <br/>
+
+                        <div id="purpose">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                            labore.
+                        </div>
+                    </div>
+                    <div class="card-action">
+                        <a href="#">PHP</a>
+                        <a href="#">Symfony2</a>
+                        <a href="#">Potatoes</a>
+                        <a href="#">Safety</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block javascripts %}
+    <script type="text/javascript">
```

Comment:
Move this script in a js file and wrap the initMap function in a class (like, AppMap for example or App.Map)